app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :body, presence: true
+  validates :user_id, :post_id, presence: true, numericality: { only_integer: true }
```

Comment:
- [ ] Kindly validate that the body is not empty. Currently, I can have a body that is an empty string, ''.
- [ ] You don't need to validate that you have a user_id or post_id, since the belongs_to keyword already takes care of that.